packages/plugin-utils/src/options.ts

```diff
@@ -261,6 +261,10 @@ export interface ResolvedOptions {
     runOnStartup: boolean
     transformers: TransformerFunction[]
     extractors: Extractor[]
+    extraTransformTargets: {
+      css: string[]
+      groups: string[]
```

Comment:
```suggestion
      detect: string[]
```

It's not only about groups, so maybe it's better to have a more general name

Reply:
True, thanks